logs/mdsd-generator.log

```diff
@@ -0,0 +1,12 @@
+2023-02-15 18:01:59.204 [main] INFO  Main - Start generating application with name: generated-application
```

Comment:
Bitte log-Dateien in die .gitignore Datei aufnehmen und diese aus dem Commit entfernen